x-pack/plugins/ml/server/models/file_data_visualizer/import_data.js

```diff
@@ -93,7 +93,7 @@ export function importDataProvider(callWithRequest) {
       body.settings = settings;
     }
 
-    await callWithRequest('indices.create', { index, body });
+    await callWithRequest('indices.create', { index, body, includeTypeName: true });
```

Comment:
I think `includeTypeName` (or is it `include_type_name`?) needs to be added to the `settings` in the `indexData` function a few lines below this.